flocker/node/functional/test_docker.py

```diff
@@ -334,11 +366,13 @@ def test_pull_image_if_necessary(self):
         """
         The Docker image is pulled if it is unavailable locally.
         """
-        image = u"busybox:latest"
+        # Use an image that isn't likely to be in use by anything, since
+        # it's old, and isn't used by other tests:
+        image = u"busybox:ubuntu-12.04"
```

Comment:
I feel like this is less likely to fail, so may as well leave it in. And we still have #1281 to actually fix the problem later on.